docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -0,0 +1,371 @@
+    "    ext_magnetic_field=(0.4, 0.0, 0.0),\n",
+    ")\n",
+    "print(f\"Hamiltonian:\\n {hamiltonian}\\n\")\n",
+    "\n",
+    "L = coupling_map.size()\n",
+    "observable = SparsePauliOp.from_sparse_list(\n",
+    "    [(\"Z\", [i], 1 / L / 2) for i in range(L)], num_qubits=L\n",
+    ")\n",
+    "print(f\"Observable:\\n {observable}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4ab867e-9d04-49f5-891a-a01e0ddc98d7",
+   "metadata": {},
+   "source": [
+    "Next you prepare the MPF. The first choice to make is whether the coefficients will be static (i.e. time-independent) or dynamic (this tutorial uses a static MPF). The next choice to make is the set of $k_j$ values. This determines how many terms will be in the MPF, as well as how many Trotter steps each term uses to simulate the time evolution. The choice of $k_j$ values is heuristic, so you need to obtain your own set of \"good\" $k_j$ values. Read guidelines on finding a good set of values at the end of the [getting started page](/guides/qiskit-addons-mpf#determine-mpf-coefficients).\n",
+    "\n",
+    "Then, once the $k_j$ values are determined, the system of equations, $Ax=b$, to solve can be prepared. The matrix $A$ is also determined by the product formula to use. The choices here are its *order*, which we set to $2$, as well as whether or not the product formula should be *symmetric*, which we set to `True`. The code snippet below selects a total time to evolve the system, the $k_j$ values to use, and the set of equations to solve using the [`qiskit_addon_mpf.static.setup_static_lse`](../api/qiskit-addon-mpf/static) method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "a90e83b3-7314-4dd2-9f9a-86fd2d5e81ca",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LSE(A=array([[1.00000000e+00, 1.00000000e+00, 1.00000000e+00],\n",
+      "       [1.56250000e-02, 6.94444444e-03, 2.77008310e-03],\n",
+      "       [2.44140625e-04, 4.82253086e-05, 7.67336039e-06]]), b=array([1., 0., 0.]))\n"
+     ]
+    }
+   ],
+   "source": [
+    "time = 8.0\n",
+    "trotter_steps = (8, 12, 19)\n",
+    "\n",
+    "lse = setup_static_lse(trotter_steps, order=2, symmetric=True)\n",
+    "print(lse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e01627b-27b8-4308-9023-4b006d73c9a5",
+   "metadata": {},
+   "source": [
+    "Once the linear system of equations has been instantiated, it can be solved either exactly or through an approximate model using a sum of squares (or the Frobenius norm for dynamic coefficients, see the [API reference](../api/qiskit-addon-mpf/costs) for more information). The choice to use an approximate model typically arises when the norm of the coefficients for the chosen set of $k_j$ values is deemed too high and a different set of $k_j$ values cannot be chosen. This guide demonstrates both to compare the results."
```

Comment:
```suggestion
    "Once the linear system of equations has been instantiated, it can be solved either exactly or through an approximate model using a sum of squares (or the Frobenius norm for dynamic coefficients; see the [API reference](../api/qiskit-addon-mpf/costs) for more information). The choice to use an approximate model typically arises when the norm of the coefficients for the chosen set of $k_j$ values is deemed too high and a different set of $k_j$ values cannot be chosen. This guide demonstrates both to compare the results."
```